src/main/java/CashDrawer.java

```diff
@@ -0,0 +1,82 @@
+public class CashDrawer {
+    // Include Cash values to build a CashDrawer Array of a Cash Register
+    int twenty;
+    int ten;
+    int five;
+    int two;
+    int one;
+    int total;
+
+    //Build out two constructors Include all properties, and a Default
+
+    public CashDrawer() {
+    }
+
+    public CashDrawer(int twenty, int ten, int five, int two, int one) {
```

Comment:
This constructor is interesting, with the implementation, what happens if you pass in negative numbers into the args? Can the register end up in an invalid state?